### ISSUE TITLE
Need more docs

### ISSUE THREAD

**Issue by D10100111001** [kelseyhightower/nocode#867]
Based on the number of issues and most of them being about usability (reaching around 700 as of now), I think the documentation is lacking. The documentation to LOC ratio is currently `Cannot divide by zero` (If you're from the JavaScript world, the ratio is `Infinity`). I think this is really low and causing a lot of problems.

Please fix ASAP.

**Comment by simonwendel** [comment]
I think that there's way too much documentation. It should perhaps be reduced until the ratio is 0/0. No docs for no code, no?

**Comment by D10100111001** [comment]
I think you may have a point if there is no documentation maybe there will be no more issues or questions.

**Comment by D10100111001** [comment]
But I think users are unable to use nocode as I'm seeing almost all issues where users accidentally type in characters for their nocode scripts. Nocode is supposed to have max and min of 0 characters but beginners are making the mistake of adding a newline character in their code causing is to become somecode. The community here at nocode is then having to answer questions that would fit the somecode repo.

**Comment by cburchert** [comment]
I agree that documentation is really a large problem. Documentation is basically always outdated, so having no documentation is obviously the superior solution and avoids confusing developers.

I recommend that we remove the description of this repository as that seems to be one of the reason of the many issues here.
